rental.rb

```diff
@@ -0,0 +1,24 @@
+require_relative './book'
+require_relative './person'
+
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Getters and Setters for `@book` and `@person` instance variables should be present